src/features/account/styles.module.scss

```diff
@@ -0,0 +1,36 @@
+.accountContainer {
+  height: 680px;
```

Comment:
COuld you use relative values here? like `em` or `%`. Also, why are we limiting this container height? Why not expanding until the last content.